src/components/LoginButton/index.tsx

```diff
@@ -5,18 +5,20 @@ import { gettext } from '../../utils';
 import styles from './styles.module.scss';
 
 type PublicProps = {
+  apiVersion: string;
```

Comment:
I guess this is only a prop for testability? It feels like it doesn't need to be a prop but the alternative of injecting `process.env` as a prop might be cumbersome.

Reply:
Exactly, I don't want to hard-code `process.env` and injecting `process.env` implies more "src" code to write (because of the TS type and `((x as any) as Env)` trick).